config/mongo.js

```diff
@@ -1,5 +1,5 @@
 import mongoose from "mongoose";
-import logger from "./logger";
```

Comment:
Is this extension needed?

Reply:
Yes, it is one of many annoying things required to use es6 modules in node.